components/amilia/actions/list-accounts/list-accounts.mjs

```diff
@@ -0,0 +1,29 @@
+import amilia from "../../amilia.app.mjs";
+
+export default {
+  key: "amilia-list-accounts",
+  name: "List Accounts",
+  description: "List all (client) accounts in an organization. [See the docs here](https://www.amilia.com/ApiDocs/v3org#GetAccounts)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    amilia,
+    showArchived: {
+      type: "boolean",
+      label: "Show Archived?",
+      description: "Option to also show archived items.",
+      optional: true,
+    },
+  },
+  async run({ $ }) {
+    const response = await this.amilia.listAccounts({
```

Comment:
I prefer to use `getAccounts` here, because is the action we are executing.

It's just a suggestion

```suggestion
    const response = await this.amilia.getAccounts({
```

Reply:
I opted to use `list-accounts` to help differentiate from `get-account`